deployments/devel/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/NVIDIA/k8s-device-plugin/deployments/devel
+
+go 1.22
+
+require github.com/matryer/moq v0.3.3
```

Comment:
0.3.4

Reply:
The plan is to have this bumped by dependabot after this merge.

Reply:
See https://github.com/NVIDIA/k8s-device-plugin/pull/809